rpa-manager/modules/ROOT/pages/processautomation-prepare-project-application.adoc

```diff
@@ -0,0 +1,111 @@
+
+A member of the process team can deploy a process on a bot that has assigned the required applications. If an application becomes non-available, the bot completes running processes using that application and restarts them when the downtime or deactivation period has ended.
+
+== Before You Begin
+
+* Ask an administrator to xref:usermanagement-manage.adoc#assign-privileges-to-a-user[assign you the needed privileges]:
++
+[cols="1,2"]
+|===
+|*Action* |*Required Privileges*
+
+|Open the *Process Automation* module
+|Process Automation Open
+
+|Create, change, and delete process applications
+|Application Create, Application Edit, Application Delete
+
+|===
+* For each application, note the timezone of the computer that it is installed on.
+
+== View All Applications
+
+To view all existing applications for automation projects, open the *Process Applications* view of the *Process Automation* module.
+
+== Create a Process Application
+
+Create one entry for each application instance required for the execution of processes.
+
+To create a process application:
+
+. Open the *Process Applications* view of the *Process Automation* module.
+. Click *Create*.
+. [[form-create-process-applications]] Complete the *Create a new application* form:
++
+* *Time Zone*:
++
+Time zone for defining downtimes.
++
+Specify a downtime in the time zone in which it occurs. If the bot that you assign the downtime to is installed on a computer that uses a different time zone, the downtime interval is converted to the bot’s time zone. If you assign the application to multiple bots in different time zones, all bots pause executing processes that use that application simultaneously. The time intervals might change, depending on daylight saving time.
++
+For example, if you specify a downtime that starts at 1:30 AM and ends at 5:30 AM in Tokyo time (JST (UTC +9)) and the bot is installed on a computer in San Francisco (PST (UTC -8)), the bot does not execute processes between 8:30 AM and 12:30 PM in San Francisco time.
++
+To avoid confusion, append the name of the time zone to the name of the downtime. For example, name your downtime `Electrical Power Outage Tokyo`.
++
+* *Downtimes*
++
+Click *Create* to add a downtime to the application.
+. Click *Save*.
+
+== Edit an Application
+
+Edit an application to change its data or down-times:
+
+. Open the *Process Applications* view of the *Process Automation* module.
+. Click *Create*.
+. Edit the *Edit the application* form.
++
+For an explanation of the properties, see  <<form-create-process-applications, *Create an Application*>>.
+. Click *Save*.
+
+== Deactivate or Activate a Process Application
+
+Deactivate an application if it is temporarily not available and you don't know when it will be available again.
+
+Use down-times to schedule maintenance and planned downtimes. Deactivation periods are not added to the list of down-times.
```

Comment:
Is down-times in this context part of the UI?

Reply:
You can make the application unavailable via the UI in two ways:
- Inside the Create/Edit Application form (you need to click a button / an icon to open the form a view and add to  the downtimes list)
- By clicking the Deactivate icon on the panel of the application in the Applications view 
The latter is described in this paragraph.

Should I add a link to the downtimes property in the creation form?

Reply:
I think I was wondering about the two different spellings in the same sentence - it would make sense to have the hyphenated version if it appeared in the UI.